<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preference</class>
 <widget class="QWidget" name="preference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="colorLabel">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="colorBox">
     <item>
      <property name="text">
       <string>Black</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yellow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Red</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_12">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>12 Hr</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_24">
       <property name="text">
        <string>24 Hr</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="formatLabel">
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
